edx_exams/apps/core/signals/handlers.py

```diff
@@ -83,3 +90,47 @@ def listen_for_exam_attempt_reset(sender, signal, **kwargs):  # pylint: disable=
         event_data={'exam_attempt': kwargs['exam_attempt']},
         event_metadata=kwargs['metadata'],
     )
+
+
+@receiver(COURSE_ACCESS_ROLE_ADDED)
+def listen_for_course_access_role_added(sender, signal, **kwargs):  # pylint: disable=unused-argument
+    """
+    Recieve COURSE_ACCESS_ROLE_ADDED signal from the event bus
+    """
+    event_data = kwargs['course_access_role_data']
+    user_data = event_data.user
+    course_key = event_data.course_key
+    role = event_data.role
+
+    if role not in COURSE_STAFF_ROLES:
+        return
+
+    user, created = User.objects.get_or_create(  # pylint: disable=unused-variable
```

Comment:
Instead of adding the pylint disable command, could you just have `user, _ = ...`, or is there a reason we should have a variable for created?